src/main/java/seedu/address/ui/HelpKeywordWindow.java

```diff
@@ -347,4 +346,19 @@ public void hide() {
     public void focus() {
         getRoot().requestFocus();
     }
+
+    /**
+     * Closes the help keyword window.
+     */
+    public void closeWindowKeyboardShortcut() {
```

Comment:
An elegant solution to enhance quality of life for users to close the help window! LGTM!